integration-tests/hibernate-reactive/src/main/resources/application.properties

```diff
@@ -2,6 +2,10 @@ quarkus.datasource.db-kind=postgresql
 quarkus.datasource.username=hibernate_orm_test
 quarkus.datasource.password=hibernate_orm_test
 
+# Hibernate config
+quarkus.hibernate-orm.log.sql=true
```

Comment:
This makes me wonder if we really shouldn't have something different for reactive than reusing the existing `quarkus.hibernate-orm` . Thoughts?

Reply:
The prefix `quarkus.hibernate-orm` is the config root for the `io.quarkus.hibernate.orm.deployment.HibernateOrmConfig` class. The "ORM" reference in that root is a bit unfortunate, but besides that it seemed to be properly organized because all of the JDBC-specific config options are isolated under the `quarkus.hibernate-orm.jdbc` prefix. So if/when we ever add reactive-only configs, we can put them under their own prefix such as `quarkus.hibernate-orm.reactive`

Reply:
ok.